swagger/sdk_api.yaml

```diff
@@ -0,0 +1,139 @@
+---
+swagger: "3.0"
+info:
+    description: "SDK API"
+    version: "1.0.0"
+    title: "SDK API"
+host: "localhost"
+basePath: "/api/v1"
+schemes:
+    - "http"
+
+paths:
+    /sdk-init-sdk:
+        post:
+            summary: "Initialize SDK"
+            requestBody:
+                required: true
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#components/schemas/SdkInitSdkRequest'
+            responses:
+                200:
+                    description: OK
+                400:
+                    description: Bad Request
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/RestErrorMessage'
+    /sdk-init-project:
+        post:
+            summary: "Initialize SDK"
+            requestBody:
+                required: true
+                content:
+                    application/json:
+                        schema:
+                            $ref: '#components/schemas/SdkInitProjectRequest'
+            responses:
+                200:
+                    description: OK
+                400:
+                    description: Bad Request
+                    content:
+                        application/json:
+                            schema:
+                                $ref: '#/components/schemas/RestErrorMessage'
+
+components:
+    schemas:
+        SdkInitSdkRequest:
+            properties:
+                data:
+                    $ref: '#/components/schemas/SdkInitSdkRequestData'
+                included:
+                    $ref: '#/components/schemas/SdkInitSdkRequestIncluded'
+
+        SdkInitSdkRequestData:
+            properties:
+                id:
+                    type: string
+                type:
+                    type: string
+                attributes:
+                    $ref: '#/components/schemas/SdkInitSdkRequestDataAttributes'
+                relationships:
+                    $ref: '#/components/schemas/SdkInitSdkRequestRelationships'
+
+        SdkInitSdkRequestDataAttributes:
+            properties:
+                developer_email:
+                    type: string
+                    example: 'developer@example.org'
```

Comment:
I guess all these params should be dynamic. it means that we need to extract all of them and create attributes. you present here only required attributes, but we can have more optional